modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -20,6 +20,8 @@ The following steps show how to integrate the {KialiProduct} with user-workload
 
 * User-workload monitoring is enabled. See link:https://docs.openshift.com/container-platform/4.16/observability/monitoring/enabling-monitoring-for-user-defined-projects.html[Enabling monitoring for user-defined projects].
 
+* OpenShift Monitoring has been configured with {SMProductShortName}.
```

Comment:
I forget, can we link to the [configuring openshift monitoring with service mesh](https://81255--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/metrics/ossm-metrics-assembly#ossm-config-openshift-monitoring-only_ossm-metrics-assembly) section here or is that against the style guide?

Reply:
That is against style. xrefs are not allowed in modules.

Reply:
An "Additional resources" section has been added with a link to "Configuring OpenShift Monitoring with Service Mesh."